operator/merkle_tree/lib/src/lib.rs

```diff
@@ -39,8 +40,39 @@ mod tests {
         let mut merkle_root = [0; 32];
         merkle_root.copy_from_slice(&hex::decode("7a3d9215cfac21a4b0e94382e53a9f26bc23ed990f9c850a31ccf3a65aec1466").unwrap());
 
-        let result = verify_merkle_tree_batch_ffi(&bytes, bytes_vec.len(), &merkle_root);
+        let result = verify_merkle_tree_batch_ffi(&bytes, bytes_vec.len() as u32, &merkle_root);
 
         assert_eq!(result, true);
     }
+
+    #[test]
+    fn test_verify_merkle_tree_batch_ffi() {
+        let addr_str = "0x66f9664f97F2b50F62D13eA064982f936dE76657";
+        let proof_generator_addr: Address = Address::parse_checksummed(addr_str, None).unwrap();
+        
+
+        let path = "./test_files/copy.json";
+
+        let mut file = File::open(path).unwrap();
+
+        let mut bytes_vec = Vec::new();
+        file.read_to_end(&mut bytes_vec).unwrap();
+
+        let mut bytes = [0; MAX_BATCH_SIZE];
+        bytes[..bytes_vec.len()].copy_from_slice(&bytes_vec);
+
+        let batch: Vec<VerificationData> = serde_json::from_slice(&batch_bytes[..batch_len as usize]).unwrap();
+        let batch_commitment = VerificationCommitmentBatch::from(&batch);
+        let batch_merkle_tree: MerkleTree<VerificationCommitmentBatch> = MerkleTree::build(&batch_commitment.0);
+        let batch_merkle_root = hex::encode(batch_merkle_tree.root);
+
+        let mut bytes = [0; MAX_BATCH_SIZE];
+        bytes[..bytes_vec.len()].copy_from_slice(&bytes_vec);
+
+        /*
+        let mut merkle_root = [0; 32];
+        merkle_root.copy_from_slice(&hex::decode("7a3d9215cfac21a4b0e94382e53a9f26bc23ed990f9c850a31ccf3a65aec1466").unwrap());
+        */
+
+    }
```

Comment:
This test is not passing. 
The name of the function is repeated so it doesn't compile. 
Also I think an assert is missing.